src/opm/input/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -150,10 +150,6 @@ namespace Opm {
                                                   this->getIOConfig(), this->getInitConfig());
         }
     }
-    catch (const OpmInputError& opm_error) {
-        OpmLog::error(opm_error.what());
-        throw;
-    } 
     catch (const std::exception& std_error) {
```

Comment:
Class `OpmInputError` [inherits](https://github.com/OPM/opm-common/blob/82226cc0b90e950edb8f3b2d44474833c55c1249/opm/common/utility/OpmInputError.hpp#L49) from `std::exception`.  Removing the `OpmInputError` clause will just mean that we end up in the `std::exception` clause instead.  That's unlikely to do any good, so we'll have to rethink this approach.

Reply:
You are correct, the wrong fix ended up in the commit, the intention was to only remove the logging call. Fixed that.